five-principles-that-guide-tidb-and-pingcap-II.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Five Principles that Guide TiDB and PingCAP (Part II)
+author: ['Max Liu']
+date: 2021-01-26
+summary: PingCAP's CEO talks about the philosophy of TiDB's evolution, focusing on two more principles.
+tags: ['Architecture']
+categories: ['Company']
+image: /images/blog/tidb-philosophy-five-principles-that-guide-tidb-and-pingcap.jpg
+---
+
+![Five Principles that Guide TiDB and PingCAP](media/tidb-philosophy-five-principles-that-guide-tidb-and-pingcap.jpg)
+
+_This article is based on a talk given by Max Liu at PingCAP Infra Meetup._
+
+In [my last post](https://pingcap.com/blog/five-principles-that-guide-tidb-and-pingcap), I talked about how we made TiDB work, made it right, and, finally, made it fast. But we didn't stop there. To build an ideal database, we need to make TiDB affordable for all and adopted by all.
```

Comment:
Could you add a link to "TiDB"? Adding internal links helps the search engine understand the website structure and improves SEO.